spec/webauthn/authenticator_assertion_response_spec.rb

```diff
@@ -26,24 +26,77 @@
     )
   end
 
+  let(:challenge) { original_challenge }
+  let(:public_key) { credential_public_key }
+  let(:sign_count) { 0 }
+  let(:user_presence) { nil }
+  let(:user_verification) { nil }
+  let(:rp_id) { nil }
+
   before do
     WebAuthn.configuration.allowed_origins = [origin]
   end
 
-  context "when everything's in place" do
+  shared_examples "a valid assertion response" do
     it "verifies" do
-      expect(
-        assertion_response.verify(original_challenge, public_key: credential_public_key, sign_count: 0)
-      ).to be_truthy
+      expect {
+        assertion_response.verify(
+          challenge,
+          public_key: public_key,
+          sign_count: sign_count,
+          user_presence: user_presence,
+          user_verification: user_verification,
+          rp_id: rp_id
+        )
+      }.not_to raise_error
```

Comment:
Why did we had to change it from `expect { ... }.to be_truthy` to  `expect { ... }..not_to raise_error`?